src/simulators/state.hpp

```diff
@@ -229,6 +229,11 @@ class Base {
   //set creg bit counts before initialize creg
   virtual void set_num_creg_bits(uint_t num_memory, uint_t num_register){}
 
+  void has_statevector_ops(bool flg)
```

Comment:
This method name looks odd. `State` does not have operations (`ops`). Maybe `enable_save_statevec` is better (though `save_amp` is also enabled).

Reply:
Hmmm, `has_statevector_ops_` is just for `AER::TensorNetwork::State`.  I think It should be a method of the class.

Reply:
I moved this to state in tensor network and changed name `enable_density_matrix`